src/js/components/stock-transfer/CreateStockTransfer.jsx

```diff
@@ -117,71 +117,45 @@ class CreateStockTransfer extends Component {
    */
   fetchStockTransferCandidates(locationId) {
     this.props.showSpinner();
-    const url = `/openboxes/api/stockTransfers?location.id=${locationId}`;
+    const url = `/openboxes/api/stockTransfers/candidates?location.id=${locationId}`;
 
     return apiClient.get(url)
-      .then(() => {
-        // TODO add after API for fetching is implemented, using mocks for testing purpose
-      })
-      .catch(() => {
-        const stockTransferItems = [{
-          _id: 1,
-          productCode: 'code2',
-          productName: 'product1',
-          lotNumber: 'lot3',
-          expirationDate: '7/11/2021',
-          zone: 'zone2',
-          binLocation: 'bin2',
-          quantityOnHand: 45,
-        }, {
-          _id: 2,
-          productCode: 'code2',
-          productName: 'product1',
-          lotNumber: 'lot2',
-          expirationDate: '7/1/2021',
-          zone: 'zone1',
-          binLocation: 'bin1',
-          quantityOnHand: 51,
-        }, {
-          _id: 3,
-          productCode: 'code2',
-          productName: 'product2',
-          lotNumber: 'lot3',
-          expirationDate: '7/22/2021',
-          zone: 'zone2',
-          binLocation: 'bin1',
-          quantityOnHand: 88,
-        }, {
-          _id: 4,
-          productCode: 'code2',
-          productName: 'product3',
-          lotNumber: 'lot2',
-          expirationDate: '7/25/2021',
-          zone: 'zone3',
-          binLocation: 'bin1',
-          quantityOnHand: 41,
-        }, {
-          _id: 5,
-          productCode: 'code2',
-          productName: 'product2',
-          lotNumber: 'lot3',
-          expirationDate: '7/2/2021',
-          zone: 'zone3',
-          binLocation: 'bin1',
-          quantityOnHand: 43,
-        }];
-
+      .then((resp) => {
+        const stockTransferCandidates = parseResponse(resp.data.data);
+        const stockTransferItems = [];
+        stockTransferCandidates.forEach((item) => {
+          // this _id is used internally in TreeTable
+          const _id = _.uniqueId('item_');
+          stockTransferItems.push({ _id, ...item });
+        });
         this.setState({ stockTransferItems }, () => this.props.hideSpinner());
-      });
+      })
+      .catch(() => this.props.hideSpinner());
   }
 
   /**
    * Sends all changes made by user in this step of stock transfer to API and updates data.
    * @public
    */
-  // TODO after API for creating stock transfer implemented
   createStockTransfer() {
-    this.props.nextPage();
+    this.props.showSpinner();
+    const url = '/openboxes/api/stockTransfers/';
```

Comment:
Did we solve the context path problem in the develop branch or was that just in the Grails upgrade branch?

Reply:
It was grails upgrade branch